src/init.rs

```diff
@@ -0,0 +1,54 @@
+use std::{io::Write, path::PathBuf};
+
+use anyhow::{bail, Result};
+use fuels::signers::wallet::Wallet;
+use termion::screen::AlternateScreen;
+
+use crate::{account::DEFAULT_WALLETS_VAULT_PATH, utils::clear_wallets_vault};
+
+pub(crate) fn init_wallet(path: Option<String>) -> Result<()> {
+    let wallet_path = match &path {
+        Some(path) => {
+            // If the provided path does not exists create it
+            std::fs::create_dir_all(path)?;
+            // If the provided path exists but used clear it
+            clear_wallets_vault(&PathBuf::from(&path))?;
+            PathBuf::from(path)
+        }
+        None => {
+            let mut path = home::home_dir().unwrap();
+            path.push(DEFAULT_WALLETS_VAULT_PATH);
+            // If the default vault path does not exists create it
+            std::fs::create_dir_all(&path)?;
+            // If the default vault path exists but used clear it
+            clear_wallets_vault(&path)?;
+            path
+        }
+    };
+    // Generate mnenomic phrase
+    let mnemonic = Wallet::generate_mnemonic_phrase(&mut rand::thread_rng(), 12)?;
```

Comment:
This should use 24 by default instead of 12. Later on it could be configurable with a parameter.